src/components/GoogleAuth/GoogleAuth.tsx

```diff
@@ -3,10 +3,12 @@ import googleIcon from 'assets/googleIcon.png';
 import { Image, Typography, Container } from 'components/GoogleAuth/GoogleAuthStyle';
 import { useTranslation } from 'react-i18next';
 
+const REACT_APP_GOOGLE_REDIRECT = 'http://localhost:3000/auth/redirect';
```

Comment:
no, it should be in env file. any link should be in env file

Reply:
I put on constants/links.ts because the google auth didn't work, looks like the function don't read the .env file